src/protocol/ipa/mod.rs

```diff
@@ -794,29 +816,29 @@ pub mod tests {
         const MAX_BREAKDOWN_KEY: u32 = 3;
         const NUM_MULTI_BITS: u32 = 3;
 
-        /// empirical value as of Mar 8, 2023.
-        const RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_3: u64 = 15453;
+        /// empirical value as of Mar 20, 2023.
+        const RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_3: u64 = 13755;
 
-        /// empirical value as of Mar 8, 2023.
-        const RECORDS_SENT_MALICIOUS_BASELINE_CAP_3: u64 = 38400;
+        /// empirical value as of Mar 20, 2023.
+        const RECORDS_SENT_MALICIOUS_BASELINE_CAP_3: u64 = 35743;
 
-        /// empirical value as of Feb 27, 2023.
-        const RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_1: u64 = 11784;
+        /// empirical value as of Mar 20, 2023.
+        const RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_1: u64 = 10086;
 
-        /// empirical value as of Feb 28, 2023.
-        const RECORDS_SENT_MALICIOUS_BASELINE_CAP_1: u64 = 29046;
+        /// empirical value as of Mar 20, 2023.
+        const RECORDS_SENT_MALICIOUS_BASELINE_CAP_1: u64 = 26389;
 
         let records: Vec<GenericReportTestInput<Fp32BitPrime, MatchKey, BreakdownKey>> = ipa_test_input!(
             [
                 { match_key: 12345, is_trigger_report: 0, breakdown_key: 1, trigger_value: 0 },
                 { match_key: 12345, is_trigger_report: 0, breakdown_key: 2, trigger_value: 0 },
-                { match_key: 68362, is_trigger_report: 0, breakdown_key: 1, trigger_value: 0 },
+                { match_key: 68363, is_trigger_report: 0, breakdown_key: 1, trigger_value: 0 },
```

Comment:
The PR description explains what's going on here... this is a hack...

Basically:
```
12345 => 0b11000000111001
68362 => 0b10000101100001010

bitwise_shares
                .into_iter()
                .enumerate()
                .fold(Replicated::ZERO, |acc, (i, bit)| {
                    acc + &(bit * F::truncate_from(1_u128 << i))
                })
```

These two inputs *both* "hash" onto Fp31::from(7)... so I'm just shifting it over to avoid the collision.